crossing-training.lic

```diff
@@ -192,6 +194,16 @@ class CrossingTraining
       pause 15
     end
   end
+  
+  def refresh_cyclic
+    return unless @settings.cyclic_training_spells
+    if @settings.cyclic_training_spells && (Time.now - @cyclic_cycle_timer > 300)
+      cyclic_skill = @settings.cyclic_cycle_skills
```

Comment:
I did add this check but as I think about it further I wonder if we wouldn't prefer to have these run while locked - to passively maintain that lock as well as afford Necro's the opportunity to list only rite of grace in their cyclic cycle and therefor keep it up at all times.